rdkit/Chem/UnitTestInchi.py

```diff
@@ -30,86 +30,26 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 from __future__ import print_function
-from rdkit.Chem import *
-from rdkit import RDConfig
-from rdkit.Chem import rdDepictor
-import unittest
+
 import gzip
-import tempfile
+import io
 import os
-import gzip
-from subprocess import Popen, PIPE
 import re
-import io
-import sys
-from rdkit.rdBase import DisableLog, EnableLog
-from rdkit.six.moves.cPickle import loads
-
-curdir = os.path.dirname(os.path.abspath(__file__))
-
-esc = chr(27)
-red = esc + '[31m'
-green = esc + '[32m'
-reset = esc + '[0m'
-
-
-class NoReentrySDMolSupplier(object):
-
-  def __init__(self, filepath, sanitize=False):
-    self.ms = SDMolSupplier()
-    self.valid = True
-    self.filepath = filepath
-    self.f = self.fileopen()
-    self.sanitize = sanitize
-
-  def fileopen(self):
-    return file(self.filepath, 'r')
-
-  def fileclose(self):
-    if self.f:
-      self.f.close()
-
-  def reset(self):
-    self.fileclose()
-    self.f = gzip.open(self.filepath, 'r')
-    self.valid = True
-
-  def next(self):
-    buf = ''
-    for line in self.f:
-      line = str(line)
-      buf += line
-      if line.strip() == '$$$$':
-        break
-    if buf:
-      self.sdf = buf
-      self.ms.SetData(buf, sanitize=self.sanitize, removeHs=False)
-      return self.ms[0]
-    else:
-      self.fileclose()
-      self.f = None
-      raise StopIteration
-
-  def dumpCurMol(self):
-    f = file(self.ms[0].GetProp('PUBCHEM_COMPOUND_CID') + '.sdf', 'w')
-    f.write(self.sdf)
-    f.close()
-    return f.name
-
-  def __iter__(self):
-    if not self.valid:
-      raise ValueError("No reentry allowed")
-    self.valid = False
-    return self
-
-  def __next__(self):
-    return self.next()
-
+import unittest
 
-class GzippedSDMolSupplier(NoReentrySDMolSupplier):
+from rdkit import RDConfig
+from rdkit.Chem import rdDepictor
+from rdkit.rdBase import DisableLog, EnableLog
+from rdkit.six.moves.cPickle import loads  # @UnresolvedImport
+from rdkit.Chem import ForwardSDMolSupplier, SanitizeMol
+from rdkit.Chem import MolFromSmiles, MolToSmiles
+from rdkit.Chem import MolFromMolBlock, MolToMolBlock
+from rdkit.Chem import INCHI_AVAILABLE, InchiReadWriteError
```

Comment:
This line and the next fail to import if INCHI_AVAILABLE is False
I think it would be better to do:
```
from rdkit.Chem import INCHI_AVAILABLE
if INCHI_AVAILABLE:
  from rdkit.Chem import InchiReadWriteError, MolToInchi, MolFromInchi, InchiToInchiKey
```